README.md

```diff
@@ -36,7 +36,7 @@ on the docs website.
 Other useful links:
  - [The official Flame site](https://flame-engine.org/).
  - [Examples](https://examples.flame-engine.org/) of most features which can be tried out from your browser.
- - [Tutorials](https://tutorials.flame-engine.org/) - A simple tutorial to get started.
+ - [Tutorials](https://github.com/RutviShukla/flame/blob/main/tutorials/space_shooter/README.md) - A simple tutorial to get started.
```

Comment:
This link should go to the tutorial page, like it was before. (Now it points to your fork...)

Reply:
The original link redirected the user to the get started page for the space shooter game.  I was just trying to make it point to the README.md for the space shooter game and provide a first-time user with more information related to the space shooter project. The original link is on the README.md for space shooter as well.

Reply:
I modified the link so that it no longer points to my fork